builtin_interfaces/Time stamp

float64 x
float64 y
float64 z
float64 yaw
float64 pitch
float64 roll

float64 acc_x
float64 acc_y
float64 acc_z
float64 acc_yaw
float64 acc_pitch
float64 acc_roll

float64 mag_x
float64 mag_y
float64 mag_z
float64 mag_strength

float64 mouse_integrated_x
float64 mouse_integrated_y
